src/oapi/vms.rs

```diff
@@ -173,6 +173,11 @@ impl Input {
                     continue;
                 }
             };
+            let tenancy = match vm.placement.clone().into_iter().next() {
```

Comment:
```rust
match vm.placement.clone().into_iter().next() {

}
```
is too complex, just match on `vm.placement`. Something like that:
```rust
match &vm.placement {
    Some(placement) => placement.tenancy.clone().unwrap_or_default(),
    None => todo!(),
}
```

Reply:
Done